### PR TITLE
add ed's tarot, eris, caroline

### PR DESCRIPTION
**Pull request by Misterio77**
A friendly fellow named [ed](https://codeberg.org/ed/) has asked me to upstream [his schemes](https://codeberg.org/ed/base16-schemes) (as he does not use github). So here they are.

### CODE REVIEW

File: caroline.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+scheme: "caroline"
+author: "ed"
```
Comment (JamyGolden) [inline caroline.yml]: Maybe add a link to Ed’s other repo or something? Not necessary but could be nice to have.

**Review by JamyGolden — APPROVED** [review]
Great 😊 added a comment about adding a link to Ed’s other repo. But I’m happy to merge if you are happy with this as is.

File: caroline.yml (hunk 1 above)
Comment (Misterio77) [reply caroline.yml]: Good ideia!

**Comment by Misterio77** [comment]
All done!

**Comment by belak** [comment]
Merged! Thanks for submitting this!

**Pull request merged**